src/domain/synthetics/tokens/useTokenAllowanceData.ts

```diff
@@ -53,5 +55,6 @@ export function useTokensAllowanceData(
 
   return {
     tokensAllowanceData: isNativeToken ? defaultValue : data,
+    mutate,
```

Comment:
What if we consider giving a more meaningful name instead of `mutate`?

Reply:
named refetchTokensAllowanceData